conftest.py

```diff
@@ -0,0 +1,4 @@
+import sys
+import os
+
+sys.path.append(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
Pathlib here thx 🙏

Reply:
It's such a small detail in a config file that it really doesn't matter.  os library comes with all python installations and this py file won't be used by anything else than the pytest. I would understand if it was used by other scripts such as we do in launcher.
But sure, I will fix it.